src/Tools/SarifValidator/ConsoleUtil.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Globalization;
+
+namespace Microsoft.CodeAnalysis.Sarif.SarifValidator
+{
+    internal static class ConsoleUtil
+    {
+        internal static void WriteError(string message, params object[] args)
+        {
+            WriteColor(ConsoleColor.Red, message, args);
```

Comment:
in my experience colorized console output is not a great idea. Will your green results show up well against a black background? What about people (like me) who uses several different color schemes, for readability and to also keep track of different cmd environments (my CBT shell looks different than my VS env, for example). Also, someone with red/green color blindness will not benefit from this choice. Just my 2 cents on the topic.